dashboard/app/views/home/_privacy.md

```diff
@@ -191,9 +191,9 @@ In particular, we do not share any Personal Information you provide with our don
 
 If your use of the Code.org Website is in the context of a partnership with your school or school district, we may allow the school or school district to access the same Student progress report that is also shared with your Teacher(s).
 
-### We may share teacher information with the teacher's training facilitators, Regional Partner, and school district
+### We may share teacher information with the teacher's training facilitators, Regional Partner, and school district, as well as other partners
```

Comment:
Capitalize Teacher?

Wording seems kinda funny. How about "We may share Teacher information with the Teacher's facilitator(s), Regional Partner, school district, or other partners"

Reply:
I think we want this lowercase b/c the teacher might not have an account yet? And the reason it's phrased like that is because the training facilitators, Regional Partner and school district are all those of the teacher whereas partners aren't the "teacher's partners." Does that make sense?